fix issue 0001 formatting:

gchartier opened the pull request:
- change date frontmatter field to published
- made fields into strings
- fixed trigger, status, initial publish date formatting

Christian-MK commented:
Closing after discussion in weekly team meeting. These contextual citations will be added to the readme where they wont complicate the use of the template

gchartier committed:
fix issue 0001 formatting:
- change date frontmatter field to published
- made fields into strings
- fixed trigger, status, initial publish date formatting
